spt/features/camera.cpp

```diff
@@ -890,6 +894,14 @@ void Camera::HandleCinematicMode(bool active)
 	currentCam = InterpPath(currentTime);
 }
 
+void Camera::HandleTraceMode(bool active)
+{
+	if (!active)
+		return;
+	extern bool SptGetActiveTracePos(Vector & pos, QAngle & ang, float& fov);
```

Comment:
Include the header preferably rather than declaring the function here separately

Reply:
Alright, I declared all cvars in tr_config